recipe/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "Pyfhel" %}
+{% set version = "3.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ibarrond/Pyfhel/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 37825ce03a54e7ca9e2d39f6fd4bf19e4d58ec28cd856fe4d7a0fef4cefed5bf
+  patches:
+    - patches/0001-no-system-include.patch
+    - patches/0002-unvendor-seal.patch
+
+build:
+  number: 0
+  # osx gives a python Abort trap: 6 on tests, for more info
+  # have a look at:
+  # https://github.com/AnacondaRecipes/pyfhel-feedstock/pull/1
+  skip: true  # [osx]
```

Comment:
```suggestion
```
@lorepirri Could you try to build on osx too?

Reply:
sure @skupr-anaconda !
I'm building locally now, but it might be worth directly also on Prefect

Reply:
the local build fails for the same reason

Reply:
I will skip `osx` again